reddit/users/api/urls.py

```diff
@@ -1,13 +1,16 @@
 from django.urls import path
-from .views import UserListView, UserDetailView, UserPostsView, EmailSignupView, UserDetailsUpdateView
+from reddit.users.api.views import UserListView, UserDetailView, EmailSignupView, UserDetailsUpdateView, \
+    RequestForgotUsernameMailView, LogoutView
 from rest_framework_simplejwt.views import TokenRefreshView, TokenObtainPairView
 
 urlpatterns = [
     path('signup/email', EmailSignupView.as_view(), name='email signup'),
     path('profile/edit', UserDetailsUpdateView.as_view(), name='edit profile'),
```

Comment:
No need to use `edit` as using `PUT` or `POST` would imply the same.

Also if you are editing a `user` object and not a `profile` object it should say `user`